elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -6,10 +6,13 @@ $pfe-modal--breakpoint--medium: 640px;
 
 $LOCAL-VARIABLES: (
 	context: light,
+	MaxHeight: 90vh,
 	MaxWidth: 70vw,
-	MinWidth: 50vw,
 	MaxWidth--mobile: 94vw,
-	MaxHeight: 90vh,
+	MaxWidth-small: 35rem,
+	MaxWidth-medium: 52.5rem,
+	MaxWidth-large: 70rem,
```

Comment:
For variants, we try to use double dashes between the property and the variant name.

```
MaxWidth--small
```

Reply:
We also need to account for the "gutter" or margin between the edge on the modal and the browser.  

![Screen Shot 2021-08-06 at 1 13 18 PM](https://user-images.githubusercontent.com/3428964/128547816-6b83fd01-de7b-4fb6-91f5-5ddce2d00df7.png)


On mobile it's fine because it's using `94vw` but the variants use hard rem units so the gutter dissapears. I think `min()` might be a good option here.


```scss
	MaxWidth-small: #{"min(35rem, 94vw)"},
	MaxWidth-medium: #{"min(52.5rem, 94vw)"},
	MaxWidth-large: #{"min(70rem, 94vw)"},
```

Note: we have to use sass interpolation to use CSS functions in Sass Maps.  I hope that makes sense :😀